remove_package

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
scope creep: does this script have sufficient knowledge to generate the `koji` command for us?

Reply:
I think it should be done in 2 steps. This is to create a PR. Once it's merged, it does indeed need to be cleaned.

What I do today is run `./rel-eng/compare-with-tags` and let that tell me which packages are unexpected and modify that.

Right now it outputs:
````
# Packages not expected in foreman-plugins-nightly-el8
 * [x] yggdrasil
```shell
koji remove-pkg foreman-plugins-nightly-el8 yggdrasil
```

# Packages missing in tag foreman-client-nightly-el9
 * [x] jsoncpp
 * [x] qpid-proton
```shell
koji add-pkg --owner=kojiadm foreman-client-nightly-el9 jsoncpp qpid-proton
```

# Packages not expected in foreman-client-nightly-rhel7
 * [x] gofer
```shell
koji remove-pkg foreman-client-nightly-rhel7 gofer
```
````

Ideally we would clean this up but I have no idea what the status is for those packages.

Right now I run `koji untag-build --all && koji-block-pkg && koji remove-pkg` but I feel that's redundant. Is `koji remove-pkg` sufficient? If so, the script you're asking for already exists.

Reply:
perfect, I totally forgot about the compare thing